scout/server/blueprints/institutes/templates/overview/cases.html

```diff
@@ -27,7 +27,7 @@
     <div class="row">
         <div class="col-md-3 mb-3">
             {{ form.case.label(class="form-label") }}
-            {{ form.case(class="form-control", placeholder="example:18201") }}
+            {{ form.case(class="form-control", placeholder="example:18201", pattern="[^\\\<\>\?\!\=\/]*", title="Characters \<>?!=/ are not allowed") }}
```

Comment:
Hm, I wonder if we needed it for the other query fields as well?

Reply:
Fix, thanks. Better to be safe than sorry!